### PR TITLE
build: add gosec scan to Travis

### PR DESCRIPTION
**Pull request by pyrooka**
This PR adds a security scan ([gosec](https://github.com/securego/gosec)) to the Travis CI pipeline and updates the golangci-lint version.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ VDIR=v5
 
 COV_OPTS=-coverprofile=coverage.txt -covermode=atomic
 
-all: testcov lint tidy
+all: testcov lint scan-gosec tidy
```
Comment (padamstx) [inline Makefile:7]: I don't think we should include scan-gosec in the "all" target, to be honest.
Let's leave it as a separate target and then we can add a `make scan-gosec` command to the script section.  Would that work?

**Review by padamstx — CHANGES_REQUESTED** [review]
I think it would be good to keep the "scan-gosec" target/goal separate from the "all" target.

File: Makefile (hunk 1 above)
Comment (pyrooka) [reply Makefile]: Yep, that makes sense.

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**